javascript/ql/src/semmle/javascript/frameworks/PropertyProjection.qll

```diff
@@ -65,6 +65,59 @@ module PropertyProjection {
 
 deprecated class CustomPropertyProjection = PropertyProjection::Range;
 
+/**
+ * Gets a callee of a simple projecty projection call.
```

Comment:
```suggestion
 * Gets a callee of a simple property projection call.
```

Reply:
Thanks. I amended your suggestions to the existing commit.